cli/flags_test.go

```diff
@@ -56,7 +56,7 @@ func TestNoLinkTesting(t *testing.T) {
 		}
 
 		for _, imp := range pkg.Imports {
-			// https: //github.com/golang/tools/blob/master/refactor/importgraph/graph.go#L115
+			// https://github.com/golang/tools/blob/master/refactor/importgraph/graph.go#L115
```

Comment:
Ah, I thought this code looked familiar but I grepped for the wrong name. We should put both forbidden-import tests in the same place (`testutil`?)` instead of duplicating this code.